Makefile

```diff
@@ -0,0 +1,16 @@
+SPARK_IMAGE_NAME ?= mesosphere/spark-2.4.4-bin-hadoop2.7-k8s
+SPARK_ON_K8S_OPERATOR_IMAGE_NAME ?= mesosphere/spark-on-k8s-operator
+SPARK_ON_K8S_DOCKER_FILE_PATH ?= images/operator/Dockerfile
+
+.PHONY: spark-build
+spark-build:
+	docker build -t ${SPARK_IMAGE_NAME} images/spark
```

Comment:
do we want to `echo` the resulting image name to a file and use that file in other targets?